mix.exs

```diff
@@ -42,11 +42,11 @@ defmodule Scrivener.Ecto.Mixfile do
   defp deps do
     [
       {:scrivener, "~> 2.0"},
-      {:ecto, "~> 2.0"},
+      {:ecto, "~> 2.1"},
       {:dialyxir, "~> 0.4.0", only: :dev},
       {:earmark, ">= 0.0.0", only: :dev},
       {:ex_doc, "~> 0.14.0", only: :dev},
-      {:postgrex, "~> 0.11.0 or ~> 0.12.0", optional: true}
+      {:postgrex, "~> 0.13.0", optional: true}
```

Comment:
We can't make this more restrictive without bumping a major version. This should change to `~> 0.11.0 or ~> 0.12.0 or ~> 0.13.0`.